ui/app/pages/send/send.component.js

```diff
@@ -209,6 +225,7 @@ export default class SendTransactionScreen extends PersistentForm {
           updateGas={(updateData) => this.updateGas(updateData)}
           scanQrCode={_ => this.props.scanQrCode()}
           showHexData={showHexData}
+          editingTransactionId={editingTransactionId}
```

Comment:
I don't think we need to pass `editingTransactionId` here

Reply:
nice catch!